tools/POTools/Services/Extracting/ResourceTextExtractor.cs

```diff
@@ -14,6 +15,11 @@ public IEnumerable<LocalizableTextInfo> Extract(string content, CancellationToke
         using (var reader = new StringReader(content))
             resourceReader = new ResXFileReader(reader);
 
-        return resourceReader.Select(kvp => new LocalizableTextInfo { Id = kvp.Value, Comment = kvp.Key });
+        // Select name, value, and comment from the resource reader.
+        return resourceReader.Select(item => new LocalizableTextInfo {
```

Comment:
nit: formatting

```suggestion
        return resourceReader.Select(item => new LocalizableTextInfo
        {
```

Reply:
Sure. I didn't want to reformat everything, to isolate the changes to as few lines as possible, and the brace was on the same line in the original so it ended up like that...

Reply:
I prefer consistent code formatting over small number of changed lines :)